### ISSUE TITLE
training with acid

### ISSUE THREAD

**Issue by ChengLiDuoJi** [donydchen/mvsplat#73]
屏幕截图 2024-10-27 215743
I use acid too. I found that after 60k steps, it often skip bad samples. Should I stop training? But I found your config file wrote that mvsplat should train 300k steps.

**Comment by ChengLiDuoJi** [comment]
![屏幕截图 2024-10-27 215743](https://github.com/user-attachments/assets/fa47cdcb-d229-4078-891f-4263bd310109)

**Comment by donydchen** [comment]
Hi @ChengLiDuoJi, the skipping operation is as expected, and there is nothing to be worried about. This is because the training mainly intends to use raw data with image resolution `360x640`, those raw images with other resolutions will be automatically skipped. Related technical details are in  https://github.com/donydchen/mvsplat/blob/31ac68880f93c40ea9e7b2f9d6ff6caf187e5871/src/dataset/dataset_re10k.py#L156-L164 

In short, ignore this warning message and continue training to 300K steps.
